tests/unit/test_project.py

```diff
@@ -251,6 +253,41 @@ def test_project_title_empty_invalid_name(yaml_loaded_data):
     assert "Invalid name for ROCK" in str(err.value)
 
 
+@pytest.mark.parametrize("entrypoint_service", [""])
+def test_project_entrypoint_service_empty(yaml_loaded_data, entrypoint_service):
+    yaml_loaded_data["entrypoint-service"] = entrypoint_service
+    project = Project.unmarshal(yaml_loaded_data)
+    assert project.entrypoint_service == entrypoint_service
```

Comment:
this is weird. this shouldn't pass. "" is not a valid service.

Reply:
An empty entrypoint_service will be considered the same as None.
Do you want to change that behavior?

Reply:
I see.

Well, I guess both approaches could be right.

My gut feeling is to be conservative and not create an interface we would regret in the future. So maybe it is safer to not allow empty values for `entrypoint-service`.

Reply:
https://docs.pydantic.dev/latest/api/types/#pydantic.types.constr